inference-engine/src/transformations/src/transformations/op_conversions/convert_interpolate1_to_interpolate4.cpp

```diff
@@ -19,50 +19,59 @@ ngraph::pass::ConvertInterpolate1ToInterpolate4::ConvertInterpolate1ToInterpolat
     MATCHER_SCOPE(ConvertInterpolate1ToInterpolate4);
```

Comment:
I see this pass is disabled by default and no one turns it on. So I suggest to remove it form pipeline until the first real usage in plugin.

Reply:
Fixed.

Reply:
Hmm, I don't see any changes in CommonOptimizations.